mitodl/mitxpro
# #407 Slick dot should not appear when no scroll

#### Pre-Flight checklist


#### What are the relevant tickets?
Fixes #407 

#### What's this PR do?
Fixes all carousels to not display the dots (paging indicator) under the carousel unless there are more slides/cards to show than are displayed together on the screen.

#### How should this be manually tested?
Set up a image/testimonial/faculty carousel with different number of cards. Change viewport width and confirm that the paging indicator dots only show up when the slider can scroll.
